docs-new/docs/graphql/core/actions/derive.mdx

```diff
@@ -164,58 +142,41 @@ Note
     any object type fields in the original output type will be dropped
     in the derived output type.
 
-</div>
+:::
 
 ## Generate handler code for a derived action
 
 For a derived action, Hasura can generate the relevant handler code to
 delegate the action back to the original operation.
 
-<div class="rst-class">
-
-api_tabs
-
-</div>
 
-<div class="tabs">
+<Tabs className="api-tabs">
+<TabItem value="console" label="Console">
 
-<div class="tab">
-
-Console
 
 Head to the `Actions -> [action-name] -> Codegen` tab in the console
 
 You can select the framework of your choice to get the corresponding
 handler boilerplate code.
 
-<div class="thumbnail" alt="Console derived action codegen">
-
-/img/graphql/core/actions/actions-derive-codegen.png
-
-</div>
-
-<div class="note">
+<Thumbnail
+  src="/img/graphql/core/actions/actions-derive-codegen.png"
+  alt="Console derived action codegen"
+/>
 
-<div class="title">
-
-Note
-
-</div>
+:::info Note
 
 The information about derived actions are stored locally on the browser
 and hence it is currently only possible to generate the delegation code
 from the browser where the action was created.
 
-</div>
-
-</div>
+:::
 
-<div class="tab">
-
-CLI
+</TabItem>
+<TabItem value="cli" label="CLI">
 
 You will have to set up codegen in the CLI first to do this as explained
-in `actions-codegen-execute`
+in [actions-codegen-execute](/graphql/core/actions/codegen/index.html#actions-codegen-execute)
```

Comment:
Broken link - Link label mismtach and the extension is html

label - `Generating handler code for your action` - this is acti=ually the missed id header section in prev comment